src/components/Main.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+    <div class="container">
+        <div class="section">
+            <div class="columns">
+                <div class="column is-8 is-offset-2">
+                    <span class = "wrapper">
+                        <div class = "readoutContainer" v-bind:style ="{float: floatVal, width: widthVal}">
+                            <div class="readout">
+                                <div v-if="!rendered">
+                                    <textarea
+                                        v-model="inputText"
+                                        class="textarea text-input"
+                                        placeholder="Paste your letter of recommendation here."
+                                    ></textarea>
+                                    <div class="btn-container">
+                                        <button
+                                            class="button is-info is-fullwidth submit-button"
+                                            v-on:click="renderIssues" 
+                                            @click = "showSidebar(); hideHeader(); changeWidth()"
+                                        >
+                                            Submit
+                                        </button>
```

Comment:
Can the formatting of these three lines be tightened up to 1-2 lines? Or is this what the linter wants?

Reply:
this is probably linter-preferred (to avoid long lines or carriage-returns inside a single attribute). but we probably shouldn't have `v-on:click` and `@click` on the same element (these are ostensibly doing the same thing!!)

 can we add `showSidebar(); hideHeader(); changeWidth()` to `renderIssues`? And one step better: can we make these conditional upon element state so that we don't need to explicitly change them? Such is the Vue'y way.

Reply:
@j6k4m8 can you expand more on the "step better" option because I thought about that, but probably I couldn't do it with an elegant implementation?